wallet-api/pom.xml

```diff
@@ -116,7 +116,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       <plugin>
         <groupId>org.projectlombok</groupId>
         <artifactId>lombok-maven-plugin</artifactId>
-        <version>1.18.0.0</version>
+        <version>1.18.20.0</version>
```

Comment:
Should this version be in maven-depmgt-pom ?

Reply:
nope, it should goes to `maven-parent-pom` since it's a maven plugin, but it's not a problem anymore anyway, since it has been removed completely in the second commit (`feat: `)